misc/generate_abi.sh

```
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+build_dir="${1}"
+
+if [[ -z "${build_dir}" ]]
+then
+    echo "Usage: $0 <build-dir>"
+    exit 1
+fi
+
+if ! $(which abidw > /dev/null 2>&1)
```

Shellcheck (0.11.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)